arch/inst/C/c.slli.yaml

```diff
@@ -11,16 +11,24 @@ definedBy:
   anyOf:
     - C
     - Zca
-base: 64
 assembly: xd, shamt
 encoding:
-  match: 000-----------10
-  variables:
-    - name: shamt
-      location: 12|6-2
-    - name: rd
-      location: 11-7
-      not: 0
+  RV32:
+    match: 0000----------10
+    variables:
+      - name: shamt
+        location: 6-2
+      - name: rd
+        location: 11-7
+        not: 0
```

Comment:
I don't think "not: 0" is correct here. Since this is a compressed instruction, a "0" register index means `x8`.

Same for the RV64 stanza, below.